app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -81,6 +81,21 @@ def redirect_to_correct_vendor(vendor, in_hybrid_mobile)
       redirect_to correct_path
     end
 
+    def capture_app_analytics(timer)
+      document_request_body = JSON.parse(@document_request.body, symbolize_names: true)[:config]
+      log_extras = {
+        vendor: 'Socure',
+        reference_id: @document_response.to_h[:referenceId],
+        vendor_request_time_in_ms: timer.results['vendor_request'],
+        success: !@url.nil?,
+        document_type: document_request_body[:documentType],
+      }
+      @document_response.to_h.merge(log_extras).merge(analytics_arguments).
+        merge(document_request_body).except(
+          :referenceId, :documentType, :extra
+        ).compact
+    end
```

Comment:
```suggestion
      analytics.idv_socure_document_request_submitted(**{attributes above})
    end
```

what are your thoughts on calling the analytics here instead and passing in document_request and document_response? 🤔 
```
def track_document_request_event(document_ request:, document_response:, timer:)
```